style.css

```
@@ -234,3 +230,135 @@ h1 {
     gap: 69px;
     padding-top: 200px;
 }
+
+/*Tablet 768px~1199px */
+
+@media (max-width: 1199px) {
+    .header {
+        padding: 0 24px;
+        z-index: 1;
+    }
+    .headline-wrap {
+        height: 776px;
+        padding-top: 84px;
+    }
+    .headline {
+        flex-direction: column;
+        gap: 211px;
+    }
+    .headline-heading br {
+        display: none;
+    }
+    .headline-heading {
+        height: 56px;
+        font-size: 36px;
+    }
+    .feature {
+        padding: 52px 24px;
+        flex-direction: column;
+        gap: 50px;
+    }
+    .feature img {
+        width: 100%;
+    }
+
+    .feature-title {
+        font-size: 32px;
+        line-height: 44.8px;
+    }
+    .feature-title br {
+        display: none;
+    }
+
+    .feature-description {
+        font-size: 18px;
+        line-height: 21.6px;
+    }
+    .feature:nth-child(2) {
+        flex-direction: column-reverse;
+    }
+    .banner {
+        height: 927px;
+    }
+    .banner-content {
+        flex-direction: column;
+        gap: 217px;
+        padding-top: 316px;
+    }
+    footer {
+        padding: 64px 104px 108px 32px;
+    }
+}
+
+/* Mobile: 375px~767px */
+
+@media (max-width: 767px) {
+    .header {
+        padding: 0 16px;
+    }
+    .headline-wrap {
+        padding-top: 48px;
+        height: 680px;
+    }
+    .headline {
+        padding-top: 42px;
+        gap: 198px;
+        height: 540px;
+    }
+    .headline-content {
+        gap: 18px;
+        height: 156px;
+    }
+    .headline-heading {
+        width: 240px;
+        height: 90px;
+        font-size: 32px;
+        line-height: 140%;
+        text-align: center;
+        color: #374151;
+    }
+    .headline-heading br {
+        display: flex;
+    }
+    .headline-button {
+        font-size: 18px;
+        width: 240px;
+        height: 48px;
+        padding: 12px;
+    }
+    .feature {
+        padding: 52px 16px;
+    }
+    .feature-tag {
+        height: 26px;
+        font-size: 16px;
+    }
+    .feature-title {
+        height: 32px;
+        font-size: 24px;
+    }
+    .feature-description {
+        height: 52px;
+        font-size: 16px;
+    }
+    .banner {
+        height: 540px;
+    }
+    .banner-content {
+        flex-direction: column;
+        gap: 131px;
+        padding-top: 121px;
+    }
+    .panda-image {
+        height: 198px;
+        width: 100%;
+    }
```

❗️ 수정요청
이미지 태그에 자체적으로 height, width 속성이 있어 이를 반응형 처리하기 위해 이런 방식으로 코드를 작성해주신 것 같습니다~
다만 이렇게 되면 height는 고정값이고 width는 부모 컨텐츠 영역의 100%로 늘어나게 되므로 이미지의 비율이 달라집니다.
구현할때 이미지의 비율을 유지하는 것이 중요하므로 비율을 유지할 수 있는 방향으로 수정해보세요!

```suggestion
    .panda-image {
        height: 198px;
        width: auto;
    }
```